src/main/java/com/commercetools/sync/inventories/utils/InventoryReferenceResolutionUtils.java

```diff
@@ -51,55 +50,45 @@
    *   </tbody>
    * </table>
    *
-   * <p><b>Note:</b> The {@link Channel} and {@link Type} references should be expanded with a key.
-   * Any reference that is not expanded will have its id in place and not replaced by the key will
-   * be considered as existing resources on the target commercetools project and the library will
-   * issues an update/create API request without reference resolution.
+   * <p><b>Note:</b> The {@link Channel} and {@link Type} reference should contain Id in the
+   * map(cache) with a key value. Any reference that is not available in the map will have its id in
+   * place and not replaced by the key will be considered as existing resources on the target
```

Comment:
```suggestion
   * map(cache) with a key value. Any reference, which have its id in place and not replaced by the key, it would not 
   * be found in the map. In this case, this reference will be considered as existing resources on  the target
```